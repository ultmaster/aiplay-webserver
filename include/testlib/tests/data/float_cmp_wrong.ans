23.5 59.3 62.8 r'sdfddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd' -100 -90.3