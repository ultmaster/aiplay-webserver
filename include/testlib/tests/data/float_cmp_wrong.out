23.5
59.3
62.8
0
100
-90.3